presto-hive/src/main/java/com/facebook/presto/hive/HiveMetadata.java

```diff
@@ -1143,13 +1142,6 @@ public HiveOutputTableHandle beginCreateTable(ConnectorSession session, Connecto
 
         List<PartitionUpdate> partitionUpdates = getPartitionUpdates(fragments);
 
-        if (isWritingStagingFilesEnabled(session)) {
-            stagingFileCommitter.commitFiles(session, handle.getSchemaName(), handle.getTableName(), partitionUpdates);
-        }
-        else {
-            verifyNoStagingFilesWritten(partitionUpdates);
```

Comment:
nit: do we want to keep this verification for now? If no - should we remove the `verifyNoStagingFilesWritten` method itself?

Reply:
Good point! We should still keep this verification until recoverable execution kicks in. Let me change that.

Reply:
As per offline discussion, we will remove all the checks here because #12529 will be merged shortly which will again remove the check.